contribute-to-tidb/code-style-and-quality-guide.md

```diff
@@ -38,3 +38,19 @@ func main() {
     fmt.Println(ping.Domain())
 }
 ```
+
+## Consider using values instead of pointers
```

Comment:
I'm afraid the title may encourage people to use values instead of pointers. How about: 

```
When to use value or pointer receiver
```

Reply:
I will change the title. But keep in mind that the point of this is to encourage people to use values instead of pointers since pointers get over-used.